tina/schema.tsx

```diff
@@ -124,20 +124,113 @@ export const schema = defineSchema({
           },
         },
         {
-          name: 'prev',
-          label: 'Prev',
-          type: 'string',
+          name: 'Next',
+          label: 'Next',
+          type: 'reference',
+          collections: ['doc', 'examples'],
```

Comment:
Lowercase name for consistency
```suggestion
          name: 'next',
          label: 'Next page',
          type: 'reference',
          collections: ['doc', 'examples'],
```

Reply:
✅